src/app/metadata/features/dataset-expansion-panel/dataset-expansion-panel.component.html

```diff
@@ -1,95 +1,183 @@
+              @if (samplesTissues().length > 0) {
+                <ul class="ms-6">
+                  @for (tissue of samplesTissues(); track tissue.value) {
+                    <li>
+                      <mat-chip class="small-chips mb-0.5 text-xs"
+                        ><span class="uppercase">{{ tissue.value }} </span>
+                        <strong>{{ tissue.count }}</strong></mat-chip
+                      >
+                    </li>
+                  }
+                </ul>
+              }
+              <p>
+                <strong>{{ samplesPhenotypes().length }}</strong>
+                Phenotypic Feature{{ samplesPhenotypes().length === 1 ? '' : 's' }}
+              </p>
+              @if (samplesPhenotypes().length > 0) {
+                <ul class="ms-6">
+                  @for (
+                    phenotypic_feature of samplesPhenotypes();
+                    track phenotypic_feature.value
+                  ) {
+                    <li>
+                      <mat-chip class="small-chips mb-0.5"
+                        ><span class="uppercase">{{ phenotypic_feature.value }} </span
+                        ><strong>{{ phenotypic_feature.count }}</strong></mat-chip
+                      >
+                    </li>
+                  }
+                </ul>
+              }
+            </div>
+          </div>
+        </div>
+        <div class="flex grow flex-col gap-y-8">
+          <div class="flex">
+            <div class="me-3 pt-4 text-5xl text-neutral-700">
+              <mat-icon inline>description</mat-icon>
+            </div>
+            <div>
+              <p><strong>File Summary:</strong></p>
+              <p>
+                <strong>{{ studiesSummary().count }}</strong> File{{
+                  studiesSummary().count === 1 ? '' : 's'
+                }}
+              </p>
+              @if (filesSummary().count > 0) {
+                <ul class="ps-5">
+                  @for (format of filesFormats(); track format.value) {
+                    <li class="list-disc uppercase">
+                      {{ format.value }} <strong>{{ format.count }}</strong>
+                    </li>
+                  }
+                </ul>
+              }
+            </div>
+          </div>
+
+          <div class="flex">
+            <div class="me-3 pt-4 text-5xl text-neutral-700">
+              <mat-icon inline>experiment</mat-icon>
+            </div>
+            <div>
+              <p><strong>Experiments Summary:</strong></p>
+              <p>
+                <strong>{{ experimentsSummary().count }}</strong> Experiment{{
+                  experimentsSummary().count === 1 ? '' : 's'
+                }}
+              </p>
+              <p>
+                <strong
+                  >{{ experimentsSummary().stats.experiment_methods.length }}
+                </strong>
+                Platform{{
+                  experimentsSummary().stats.experiment_methods.length === 1 ? '' : 's'
```

Comment:
It seems you forgot to use some of the custom signals like experimentSummaryStats.